fetch.bs

```diff
@@ -231,6 +231,12 @@ following <a for=struct>items</a>: [[RESOURCE-TIMING]] [[NAVIGATION-TIMING]]
  <dd>Null or a <a for=/>connection timing info</a>.
 </dl>
 
+<p>To <dfn export lt="create an opaque timing info|creating an opaque timing info">create an
+opaque timing info</dfn>, given a <a for=/>fetch timing info</a> <var>timingInfo</var>,
+ create a new <a for=/>fetch timing info</a> whose <a for="fetch timing info">start time</a> and
+ <a for="fetch timing info">post-redirect start time</a> are <var>timingInfo</var>'s
+ <a for="fetch timing info">start time</a>.
```

Comment:
Indentation is wrong in this block. Also has newlines inside phrasing elements.

Reply:
Fixed